src/V3AstNodes.h

```diff
@@ -924,7 +924,8 @@ class AstBasicDType final : public AstNodeDType {
         return keyword() == AstBasicDTypeKwd::BIT && isRanged();
     }
     bool isDpiLogicVec() const {  // DPI uses svLogicVecVal
-        return keyword().isFourstate() && !(keyword() == AstBasicDTypeKwd::LOGIC && !isRanged());
+        return keyword().isFourstate() && !(keyword() == AstBasicDTypeKwd::LOGIC && !isRanged())
+               && !(keyword() == AstBasicDTypeKwd::LOGIC_IMPLICIT && width1());
```

Comment:
```suggestion
        return keyword().isFourstate() && !((keyword() == AstBasicDTypeKwd::LOGIC
                        || keyword() == AstBasicDTypeKwd::LOGIC_IMPLICIT) && !isRanged());
```
But see intro comment.